holoviews/plotting/bokeh/plot.py

```diff
@@ -582,11 +582,13 @@ def initialize_plot(self, ranges=None, plots=[]):
             else:
                 passed_plots.append(None)
 
-        toolbar_location = None if bokeh3 else self.toolbar
```

Comment:
@Hoxbro Do you remember why this was needed?

Reply:
See [this](https://github.com/holoviz/holoviews/pull/5388#issuecomment-1305969977) comment and the next.

Reply:
Fixed the logic in https://github.com/holoviz/holoviews/pull/5640/commits/4a0b16ee21c615d831c04c6044af70c0137a2496